### ISSUE TITLE
Comment posting on Docs becomes disabled when plugin activated

### ISSUE THREAD

**Issue by FluXs** [boonebgorges/buddypress-docs-in-group#1]
Hello

I am unsure if you are looking for feedback seeing as this is alpha/wip but I have noticed when you activate this plugin, comments become disabled "Comment posting has been disabled on this doc." 

When I disable this plugin, commenting works once more on the Docs.

I spent a an hour or so looking to see where the problem is but only got as far as finding out this fires as true (from single/index.php)

<?php if ( apply_filters( 'bp_docs_allow_comment_section', true ) ) : ?>

Which goes to... single/comments.php

In there, this line fails to be true:

<?php if ( comments_open() && current_user_can( 'bp_docs_post_comments' ) ) : ?>

...and instead fails to "Comment posting has been disabled on this doc." I think specifically its the check of "comments_open()" which returns false

That is as far as I got

(Tested with latest version from Dec 12 and with BuddyPress Docs v1.8.5)

Thanks for this work done on this plugin

Regards,
Mark

**Comment by boonebgorges** [comment]
Thanks for reporting. Do you happen to know whether you're using theme 
compatibility for the plugin? I think the function is 
bp_docs_do_theme_compat(). Are you using a child theme of bp-default?

**Comment by FluXs** [comment]
Hi, that was quick :)

Looking at the function: bp_docs_do_theme_compat() I think its asking are we using buddypress theme compatibility? I have created my own theme, copying parts of the bp-legacy template into my theme when required. So I think yes I am.

But for troubleshooting I switched back to the Twenty Fifteen and Twenty Fourteen themes and I get the same issue of comment posting becoming disabled when I activate this plugin.

I should have also mentioned initially, if the doc already had a comment or two posted, those comments will still appear when this plugin is re-activated (but still with the "Comment posting has been disabled on this doc" msg)

![screenshot](https://cloud.githubusercontent.com/assets/10354009/5584772/b77a769c-908f-11e4-999c-768f8c61f1a0.png)
